common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -125,8 +125,6 @@ public enum Feature {
 
         PASSKEYS("Passkeys", Type.PREVIEW),
 
-        CACHE_EMBEDDED_REMOTE_STORE("Support for remote-store in embedded Infinispan caches", Type.EXPERIMENTAL),
```

Comment:
Should we be documenting that this feature was removed in `changes-26_3_0.adoc` file?